branding/style/branding.css

```diff
@@ -233,6 +236,9 @@
 }
 
 /* edit contents */
+#activities #btn_editfile {
```

Comment:
Hmm, I would like to hide them on "details / open" page but keep them in "edit" page. 

Can we distinguish on "column-activities"?

Reply:
Sure we can. Done.

Reply:
Can you make the same change on use_url and open?

Reply:
Ah sorry missed the "them". Done.